app/models/katello/glue/pulp/repo.rb

```diff
@@ -775,6 +775,8 @@ def full_path(smart_proxy = nil)
       scheme   = (self.unprotected ? 'http' : 'https')
       if docker?
         "#{pulp_uri.host.downcase}:5000/#{pulp_id}"
+      elsif file?
+        "#{pulp_uri.host.downcase}/pulp/repos/#{pulp_id}"
```

Comment:
Ah, okay. Pulp team told me it was published to /pulp/repos,  but I've tested the /pulp/isos and that is correct. I have updated the p-r!